packages/flutter_tools/templates/ffi_plugin/src.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,29 @@
+# We use the CMake version connected to the version of the Android Gradle Plugin.
+#
+# Android Gradle Plugin 4.1.0: Android NDK 21.1.6352462.
+# https://developer.android.com/studio/projects/install-ndk#default-ndk-per-agp
+# CMake 3.10.2.
+# https://developer.android.com/studio/projects/install-ndk#vanilla_cmake
+#
+# You are strongly encouraged to not ship FFI plugins requiring newer
+# versions of the NDK or CMake, because your downstream users will not be
+# able to use them.
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.11" CACHE STRING "Minimum OS X deployment version")
+
+project({{projectName}}_library VERSION 0.0.1 LANGUAGES C)
+
+# Using CMAKE_CURRENT_LIST_DIR so this can be included from another CMakeLists.
```

Comment:
Why are you doing this with `include` (which requires this workaround) rather than adding it via `add_subdirectory`?